db/migrate/20250625003228_create_raw_pending_column_transactions.rb

```diff
@@ -0,0 +1,13 @@
+class CreateRawPendingColumnTransactions < ActiveRecord::Migration[7.2]
+  def change
+    create_table :raw_pending_column_transactions do |t|
+      t.string :column_id
+      t.integer :column_event_type
+      t.jsonb :column_transaction
+      t.text :description
+      t.date :date_posted
+      t.integer :amount_cents
```

Comment:
- **question**: should we expect all of these columns to potentially have `NULL` values?
- **question**: should `column_id` be unique?